scripts/ansible/VM/create.yml

```diff
@@ -0,0 +1,165 @@
+---
```

Comment:
All resources are installed in the Default Resource Group. Please create a specific Resource Group (with name based on `name_prefix`) for these resources, and put all created resources into it